platform/linux-generic/m4/odp_crypto.m4

```diff
@@ -0,0 +1,33 @@
+# ODP_CRYPTO
+# ----------
+# Select default crypto implementation
+AC_ARG_ENABLE([crypto-default],
+             [AS_HELP_STRING([--enable-crypto-default],
+                             [choose default crypto implementation [openssl/armv8crypto/null] (linux-generic)])],
+             [default_crypto=$enableval], [default_crypto=openssl])
+
+AS_IF([test "x$default_crypto" != "xopenssl" -a "x$default_crypto" != "xarmv8crypto" -a "x$default_crypto" != "xnull"],
+      [AC_MSG_ERROR([Invalid crypto implementation name])])
+
+AS_IF([test "${with_openssl}" = "no"],
+      [default_crypto=null])
```

Comment:
This is not correct as it prevents using the ARM crypto library without OpenSSL. There is no reason to require openssl with the ARM crypto library. OpenSSL crypto requires OpenSSL but that this specific to OpenSSL crypto.

Reply:
I agree. I will remove this condition. Thanks